spyderlib/plugins/shortcuts.py

```diff
@@ -354,12 +361,66 @@ def setup_page(self):
         reset_btn = QPushButton(_("Reset to default values"))
         reset_btn.clicked.connect(self.reset_to_default)
         vlayout.addWidget(reset_btn)
+        self.emacs_cb = self._get_emacs_cb()
+        self.emacs_cb.clicked.connect(self.emacs_checked)
+        vlayout.addWidget(self.emacs_cb)
         self.setLayout(vlayout)
+
+    def _get_emacs_cb(self):
+        cb = QCheckBox('Use Emacs Keybindings')
+        if not CONF.has_section('keyboard_preferences'):
+            CONF.add_section('keyboard_preferences')
+        use_emacs = CONF.get('keyboard_preferences', 'use_emacs', False)
+        cb.setChecked(use_emacs)
+        tooltip = ["Set this option to use Emacs Keybindings",
+                   "Ctrl A/E, N/P and K/U are set automatically",
+                   "Select All is moved to Shift+Ctrl+A",
+                   "Forward/Backward Char/Word are not set automatically"]

```

Comment:
I was trying to minimize disruption to the existing shortcuts (like Ctrl+F), and we had no support for the true Emacs-style find.  I say we remove these two, since they're not in the IPython shortcuts.